actors/common/src/main/java/org/sunbird/learner/actors/OrganisationManagementActor.java

```diff
@@ -1665,6 +1663,18 @@ private String getRootOrgIdFromChannel(String channel) {
     return "";
   }
 
+  private Integer getStatusFromChannel(String channel) {
+    ProjectLogger.log(
+        "OrganisationManagementActor:getStatusFromChannel: channel = " + channel,
+        LoggerEnum.INFO.name());
+    if (!StringUtils.isBlank(channel)) {
+      List<Map<String, Object>> list = getOrg(channel);
+      if (!list.isEmpty()) return (Integer) list.get(0).getOrDefault(JsonKey.STATUS, 0);
```

Comment:
From here you are returning Wrapper Object and in caller u are converting to int. there might be scenario Status key available but value as NULL , so please handle it as well.